docs/getting-started/node-nvm.mdx

```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 2
+---
+
+# NVM and Node.Js
+
+## Uninstall Node.Js(if you have it installed outside of nvm)
+
+Try to look for your current node.js file in `C:\Program Files` and uninstall it.
```

Comment:
Would it be a better idea to uninstall it using control panel or similar tools to be sure that the registry entries get removed as well?

Reply:
Should I just ask them to uninstall the node.js using control panel?

Reply:
That would be ideal unless there's a better way.